packages/react/src/themes/teams/components/Tree/treeItemStyles.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
+import { pxToRem } from '../../../../lib'
+import getBorderFocusStyles from '../../getBorderFocusStyles'
+import { TreeItemProps } from '../../../../components/Tree/TreeItem'
+import TreeTitle from '../../../../components/Tree/TreeTitle'
+
+const treeItemStyles: ComponentSlotStylesInput<TreeItemProps> = {
+  root: ({ theme: { siteVariables }, props: { level } }): ICSSInJSStyle => ({
```

Comment:
```suggestion
  root: ({ theme: { siteVariables }, props: p } }): ICSSInJSStyle => ({
```

Please use `p.*` naming